src/pages/HomePage.tsx

```diff
@@ -181,14 +95,28 @@ const HomePage: FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [client]);
 
+    const onInsertLink = useCallback((sessionLink) => {
+        if (client) {
+            client?.deskpro().send({
+                type: "append_to_active_ticket_reply_box",
```

Comment:
Please use the `client.deskpro().appendLinkToActiveTicketReplyBox()` method to add the link, as per the docs: https://support.deskpro.com/en-US/guides/developers/deskpro-ui

Reply:
@zpawn  please use the session URL for both the link text and URL

Reply:
Also, don't think you need the if block if you're coalescing the call, i.e. `client?.deskpro()...`

Reply:
Everything fixed